BioForm/profiles/urls.py

```diff
@@ -6,9 +6,15 @@
     path('profiles/index/', views.index, name="home"),
     path('profiles/register/', views.register_user, name="register"),
     path('profiles/login/', views.login_user, name="login"),
-    path('profiles/create/', views.create_record, name="create"),
+    path('profiles/createbio/', views.create_record, name="create_bio"),
+    path('profiles/createblog/', views.create_blog, name="create_blog"),
     path('profiles/logout', views.logout_user,name="logout"),
     path('profiles/update/<int:id>/', views.update_record, name="update"),
-    path('profiles/delete/<int:id>/', views.delete_record, name="delete")
+    path('profiles/delete/<int:id>/', views.delete_record, name="delete"),
+    path('profiles/update-blog/<int:id>/', views.update_blog, name="update-blog"),
+    path('profiles/delete-blog/<int:id>/', views.delete_blog, name="delete-blog"),
+    path('profiles/blogs/', views.display_blogs, name="blogs"),
+    path('profiles/single-blog/<int:id>/', views.display_single_blog, name="single-blog"),
```

Comment:
Blogs shouldn't lie under profiles you can create a new base URL also go through REST protocols to define better URL names.